### ISSUE TITLE
Improve accessibility for dialog close action

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#124]
As user reports:

> Close button of A11y checker not announced by SR
> 
> The A11y checker dialog has a close button that does not receive focus. The dialog can be closed by pressing escape. This is not explained to users however, so it may not be evident. It is common for excape to be a shortcut to a 'cancel' button, which the dialog does not have. Either the close button should be made allowed to receive keyboard focus, or a cancel button should be added to the dialog which can receive focus.
### Include information to dialog description

I think that sufficient solution would be to include an information to dialogs aria description. It's a very common key to do that (also used in CKE dialogs), so most of users will have this presumption.
### Adding close button to focus list

Doable but I don't find it really that profitable. It will be a big drawback due to increasing focus list size with "button" that will be hardly ever used.

Everybody using keyboard will almost always use `ESC` key for that.
### Adding a cancel button

I'd think that adding cancel button is a big no for us as we don't want to bloat UI.
